packages/core/content-releases/admin/src/pages/ReleasesPage.tsx

```diff
@@ -1,25 +1,114 @@
 import * as React from 'react';
 
-import { Button, HeaderLayout } from '@strapi/design-system';
-import { CheckPermissions } from '@strapi/helper-plugin';
+import {
+  Box,
+  Button,
+  ContentLayout,
+  Flex,
+  HeaderLayout,
+  Main,
+  Tab,
+  TabGroup,
+  TabPanel,
+  TabPanels,
+  Tabs,
+} from '@strapi/design-system';
+import {
+  CheckPermissions,
+  LoadingIndicatorPage,
+  PageSizeURLQuery,
+  PaginationURLQuery,
+  useQueryParams,
+} from '@strapi/helper-plugin';
 import { Plus } from '@strapi/icons';
 import { useIntl } from 'react-intl';
 
 import { AddReleaseDialog } from '../components/AddReleaseDialog';
+import { ReleasesGrid } from '../components/ReleasesGrid';
 import { PERMISSIONS } from '../constants';
+import { useGetReleasesQuery, GetAllReleasesQueryParams } from '../modules/releaseSlice';
 
 const ReleasesPage = () => {
   const [addReleaseDialogIsShown, setAddReleaseDialogIsShown] = React.useState(false);
+  const [tabSelected, setTabSelected] = React.useState<'pending' | 'done'>('pending');
   const { formatMessage } = useIntl();
+  const [{ query }, setQuery] = useQueryParams<GetAllReleasesQueryParams>();
+  const response = useGetReleasesQuery(query);
 
-  const total = 0; // TODO: replace it with the total number of releases
+  const { isLoading, isSuccess, isFetching, isError } = response;
 
   const toggleAddReleaseDialog = () => {
     setAddReleaseDialogIsShown((prev) => !prev);
   };
 
+  if (isLoading || isFetching) {
```

Comment:
Whats the difference between `isLoading` and `isFetching`?

Reply:
there are situations where the isLoading is false and isFetching is true, so I want to handle the data are not ready, if you want I can just use isFetching
<img width="1065" alt="isFetching" src="https://github.com/strapi/strapi/assets/2589748/1b0b77df-c8c0-40d3-bcc3-076ae0f539cf">

Reply:
would it then be better to use `status === "pending"`? what do the docs suggest?

Reply:
in the DOCS they uses isLoading and isFetching, basically they handle the first API call with isLoading and when they refetch some data it is suggested to use isFetching to show the previous data but for example disable it in the view, https://redux-toolkit.js.org/rtk-query/usage/queries#query-loading-state

Reply:
my perception is "isFetching" can cover all the cases

Reply:
If we use `isFetching` we'll render the spinner every time we get new data so if you're on the page & you invalidate the data you'll get the spinner again, otherwise if we use `isLoading` there's no indicator that we're background re-loading the data. Might be worth checking with Lucas what he thinks?

Reply:
I will go with isLoading, it handles all my cases correctly